projects/slap_manipulation/src/slap_manipulation/dataloaders/robot_loader.py

```diff
@@ -466,6 +478,22 @@ def get_datum(self, trial, keypoint_idx, verbose=False):
         if self.annotations is not None:
             cmd_opts = self.annotations[cmd]
             cmd = cmd_opts[np.random.randint(len(cmd_opts))]
+        if self.per_action_cmd:
+            # hard-coded for pick_up_bottle right now
+            cmd = [
+                "approach-pose-action bottle",
+                "grasp-action bottle",
+                "lift-action bottle",
+            ]
+        # TODO: remove this and read from a yaml file instead
+        if "knob" in cmd:
```

Comment:
this is a bit confusing, do tyou mean handle?

Reply:
This was a small test; I meant microwave knob, I was testing different interaction points. Removed this, will read from YAML in next iteration.